src/main/java/com/github/jenkinsci/zaproxyplugin/ZAPDriver.java

```diff
@@ -1265,8 +1266,8 @@ private String setUpUser(BuildListener listener, ClientApi zapClientAPI, String
         // means that any value we add for the configuration values has to be
         // URL encoded.
         StringBuilder userAuthConfig = new StringBuilder();
-        userAuthConfig.append("username=").append(URLEncoder.encode(username, "UTF-8"));
-        userAuthConfig.append("&password=").append(URLEncoder.encode(password, "UTF-8"));
+        userAuthConfig.append("Username=").append(URLEncoder.encode(username, "UTF-8"));
+        userAuthConfig.append("&Password=").append(URLEncoder.encode(password, "UTF-8"));
```

Comment:
That's an issue, they should really be the same (for consistency), you should be able to send both pair of parameters at the same time without problems (that would work with both methods and will not stop working if/once Zest is changed to use lower case names).